tracer/test/snapshots/AspNetMvc4Tests.Classic.NoFF.__path=_Admin_Home_statusCode=200.verified.txt

```diff
@@ -53,11 +55,15 @@
     Type: web,
     Tags: {
       env: integration_tests,
+      http.client_ip: 127.0.0.1,
       http.method: GET,
       http.request.headers.host: localhost:00000,
+      http.route: /home/hellofromchild,
       http.status_code: 200,
-      http.url: /admin/home,
+      http.url: http://localhost/Admin/Home,
```

Comment:
This change is the expected one, correct? Also, just to clarify, we're supposed to capture the _original_ url right, i.e. not lowercase it?

Reply:
yes we're supposed to capture the url, with the non decoded query string, and as it's not been written to lowercase it, I assume we leave it that way 😃